spec/rubocop/cop/performance/redundant_merge_spec.rb

```diff
@@ -3,8 +3,12 @@
 
 require 'spec_helper'
 
-describe RuboCop::Cop::Performance::RedundantMerge do
-  subject(:cop) { described_class.new }
+describe RuboCop::Cop::Performance::RedundantMerge, :config do
+  subject(:cop) { described_class.new(config) }
+
+  let(:cop_config) do
+    { 'MaxKeyValuePairs' => 2 }
```

Comment:
By default they will be from `default.yml`, so it's not really necessary to set this explicitly (although this makes the test more resilient to config changes).